### PR TITLE
New package: python3-cookiecutter-2.1.1

### PR DESCRIPTION
**Pull request by tranzystorek-io**
- New package: python3-jinja2-time-0.2.0
- New package: python3-cookiecutter-2.1.1

#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

### CODE REVIEW

File: srcpkgs/python3-jinja2-time/patches/arrow-replace-to-shift.patch [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
++        shift_params = {}
+         for param in offset.split(','):
+             interval, value = param.split('=')
+-            replace_params[interval.strip()] = float(operator + value.strip())
+-        d = d.replace(**replace_params)
++            shift_params[interval.strip()] = float(operator + value.strip())
++        d = d.shift(**shift_params)
+ 
+         if datetime_format is None:
+             datetime_format = self.environment.datetime_format
```
Comment (ahesford) [inline srcpkgs/python3-jinja2-time/patches/arrow-replace-to-shift.patch:19]: Why does this patch exist?

Comment (tranzystorek-io) [reply srcpkgs/python3-jinja2-time/patches/arrow-replace-to-shift.patch:19]: jinja2-time hasn't received an update for a long time, this is purely to let it interact with new versions of python3-Arrow

Comment (ahesford) [reply srcpkgs/python3-jinja2-time/patches/arrow-replace-to-shift.patch:19]: The only two real commits in the last six years have been to switch CI providers an limit dependency versions on old Python. This looks like abandonware.

Comment (tranzystorek-io) [reply srcpkgs/python3-jinja2-time/patches/arrow-replace-to-shift.patch:19]: maybe, but it's still a dependency :/

**Comment by ahesford** [comment]
Given the status of jinja2-time, I'm loath to package it. We can revisit this if cookiecutter  drops the dependency or jinja2-time appears to come back to life.

**Pull request closed without merging**